sdcm/nemesis.py

```diff
@@ -61,6 +67,97 @@ class UnsupportedNemesis(Exception):
     """ raised from within a nemesis execution to skip this nemesis"""
 
 
+class FieldTypeDefinition():
```

Comment:
Please move helper classes out of nemesis, maybe to utils.* or test_lib/

Reply:
move to `test_lib`